crates/pallet-domains/src/tests.rs

```diff
@@ -992,6 +998,152 @@ fn generate_invalid_domain_extrinsic_root_fraud_proof<T: Config + pallet_timesta
     })
 }
 
+#[test]
+fn test_true_invalid_bundles_inherent_extrinsic_proof() {
```

Comment:
Please add a end-to-end integration test (like other fraud proof) that covers the fraud proof construct/submit/verify/processing flow